docs/catalog/cloud/review-platform-and-service-updates-regularly.md

```diff
@@ -0,0 +1,33 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Review platform and service updates regularly
+
+## Description
+Platform updates enable you to use the latest functionality and features to help increase efficiency. Running on outdated software can result in running a suboptimal workload with unnecessary performance issues. New software tends to be more efficient in general.
+
+## Solution
+Upgrade to newer and more efficient services as they become available.
```

Comment:
I'm not sure I agree with this as a pattern. Newer software isn't necessarily more efficient, in fact it goes against one of the Embodied Carbon principles of ensuring your software is backwards compatible and doesn't require new hardware. Can you provide more specificity here or connect it to a metric that supports the view of "newer == more efficient"?

Reply:
I would agree that newer software tends to be less efficient in most of the cases (I might be biased, as I need to buy a new laptop and upgrade from my 2016 one... :/ )

@yelghali : I think it would be a good idea to test/validate if the newer software is more efficient before upgrading or provides better support for your use case. When not, then skip